trimmer_decor.rb

```diff
@@ -0,0 +1,8 @@
+require_relative './base_decorator'
+
+# TrimmerDecorator class to trim names to max of 10
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.strip.slice(0, 10) if @nameable.correct_name.length > 10
```

Comment:
- Please, kindly make the range (0, 9) as your range will return 11 characters instead of 10 characters.